catalog/MDCDragons.xcodeproj/xcshareddata/xcschemes/MDCDragons.xcscheme

```diff
@@ -690,12 +690,22 @@
             skipped = "NO">
             <BuildableReference
                BuildableIdentifier = "primary"
-               BlueprintIdentifier = "CCD06113384824F0016FA0CF5D813CF6"
+               BlueprintIdentifier = "2E500851D3F02492C5E3B01ED09B8D28"
```

Comment:
Do you know why these identifiers all changed?

Reply:
I'm not sure - that's what it did when I regenerated from the podfile.

Reply:
Strange; perhaps it's due to a new Cocoapods version? Not blocking so carry on!

Reply:
I'm not using any new CocoaPods version.  [We're pinned to CocoaPods version 1.6.1](https://github.com/material-components/material-components-ios/tree/develop/docs/build-env#cocoapods).

Reply:
This seems likely just to do with Xcode deciding to regenerate the identifiers and not something we should block this PR on.